aptos-move/e2e-benchmark/src/main.rs

```diff
@@ -117,10 +117,10 @@ fn main() {
             num_objects: 100,
             object_payload_size: 10 * 1024,
         }),
-        (66, EntryPoints::InitializeVectorPicture { length: 40 }),
+        (61, EntryPoints::InitializeVectorPicture { length: 40 }),
         (16, EntryPoints::VectorPicture { length: 40 }),
         (16, EntryPoints::VectorPictureRead { length: 40 }),
-        (28299, EntryPoints::InitializeVectorPicture {
+        (23256, EntryPoints::InitializeVectorPicture {
```

Comment:
Do we need to update executor-benchmark numbers as well? Seems like a regression

Reply:
this is a perf improvement, as this is in microseconds

Reply:
Ah, awesome then